### PR TITLE
Icon list changed with link #225

### PR DESCRIPTION
**Pull request by MadhaviMandalia**
## Proposed changes
Removed icon list from the page and change the text with the relevant link which is `Search Google's Icons`

## Screenshots:
img

## Types of changes

- [x] Bug fix (non-breaking change which fixes an issue).
- [ ] New feature (non-breaking change which adds functionality).
- [ ] Breaking change (fix or feature that would cause existing functionality to change).

## Checklist:

- [x] I have read the **[CONTRIBUTING document](https://github.com/materializecss/materialize/blob/main/CONTRIBUTING.md)**.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.

### CODE REVIEW

File: pug/page-contents/icons_content.html [hunk 1]
```diff
@@ -35,7 +35,7 @@ <h5>Sizes</h5>
             </code></pre>
 
             <!-- Icon Listing -->
-            <div class="row icon-container">
+            <!-- <div class="row icon-container">
```
Comment (DanielRuf) [inline pug/page-contents/icons_content.html:38]: Please delete the whole part, we don't need it anymore =)

Comment (DanielRuf) [inline pug/page-contents/icons_content.html:38]: Please delete the whole part, we don't need it anymore =)

File: pug/page-contents/icons_content.html [hunk 2]
```diff
@@ -3,7 +3,7 @@
     <div class="col s12 m8 offset-m1 xl7 offset-xl1">
 
     <div id="usage" class="scrollspy">
-            <p class="caption">We have included 932 Material Design Icons courtesy of Google. You can download them directly from the <a href="https://google.github.io/material-design-icons/#icon-font-for-the-web">Material Design specs</a>.</p>
+            <p class="caption">We use Google Material Icons by Google. They provide a searchable list here (which we do not include in the documentation here), which will also show you the relevant icon names for the CSS classes. You can download the icons directly from the <a href="https://google.github.io/material-design-icons/#icon-font-for-the-web">Material Design specs</a>.</p>
```
Comment (DanielRuf) [inline pug/page-contents/icons_content.html:6]:
```suggestion
            <p class="caption">We use Google Material Icons by Google. They provide a searchable list (which we do not include in the documentation here), which will also show you the relevant icon names for the CSS classes. You can download the icons directly from the <a href="https://google.github.io/material-design-icons/#icon-font-for-the-web">Material Design specs</a>.</p>
```

**Review by DanielRuf — CHANGES_REQUESTED** [review]

File: pug/page-contents/icons_content.html (hunk 2 above)
Comment (MadhaviMandalia) [reply pug/page-contents/icons_content.html]: Committed the suggested changes please let me know if there is any other change.

Regards.

**Comment by DanielRuf** [comment]
@MadhaviMandalia all fine so far. But commitlint check does not succeed.

Please change the commit message from

> docs:Icon table removed, rephrase the Icons description #225

to

> docs: remove icon table, rephrase the icons description #225

**Comment by MadhaviMandalia** [comment]
@DanielRuf Thanks 😃

**Comment by DanielRuf** [comment]
@MadhaviMandalia thanks for your contribution to materializecss.
Welcome to the materializecss project.

I have sent you an invite as member of https://github.com/materializecss. Also there is just one small change needed:
https://github.com/materializecss/materialize/commit/55ccfa4bb1b7c75610d3afa0c10d53c64d85e67d#commitcomment-62917089

Let us know if you want to do this small change or if someone else can do it.

**Comment by MadhaviMandalia** [comment]
Hello @DanielRuf ,
I will raise another PR for this issue. Where should I put the link on the page? 

Thanks & Regards.

**Comment by DanielRuf** [comment]
@MadhaviMandalia 

> We use Google Material Icons by Google. They provide a **searchable list** (which we do not include in the documentation here), which will also show you the relevant icon names for the CSS classes. 

You can link these two words like

> <a href="...">searchable list</a>

**Comment by MadhaviMandalia** [comment]
Oops I thought we had to implement this somewhere on the tutorial or something I didn't look at the description 😄

**Pull request merged**